client/profile.html

```diff
@@ -69,7 +69,7 @@ <h3 class="username">{{user.profile.name}}</h3>
           <h4 class="list-group-item-heading">My Latest Tweets</h4>
         </li>
 
-        {{> tweets}}
+        {{> tweets tweets=profileTweets}}
```

Comment:
Oh wow, so you are overridding the helper call to tweets and it runs profileTweets instead? Didn't even realize you could do that with spacebars/blaze